client/my-sites/google-my-business/stats/index.js

```diff
@@ -65,9 +66,39 @@ class GoogleMyBusinessStats extends Component {
 		} );
 	};
 
+	renderPlaceholder() {
+		const { siteId, siteSlug, translate } = this.props;
+
+		return (
+			<Main wideLayout>
+				<PageViewTracker
+					path="/google-my-business/stats/:site"
+					title="Google My Business > Stats"
+				/>
+
+				<DocumentHead title={ translate( 'Stats' ) } />
+
+				<SidebarNavigation />
+
+				<StatsNavigation selectedItem={ 'googleMyBusiness' } siteId={ siteId } slug={ siteSlug } />
+
+				<QuerySiteSettings siteId={ siteId } />
+				<QueryKeyringConnections />
+
+				<div className="gmb-stats__metrics">
+					<div className="gmb-stats__metric" />
+				</div>
+			</Main>
```

Comment:
We're duplicating lots of code here. Could we find a way to avoid that?

Reply:
Yes that needs more work 👍